File: fuelnet_back/src/main/java/com/fuelnet/fuelnet/dto/FuelTankResponse.java
package com.fuelnet.fuelnet.dto;

import com.fuelnet.fuelnet.enums.FuelType;
import lombok.Builder;
import lombok.Data;
import java.math.BigDecimal;
import java.time.LocalDateTime;

@Data
@Builder
public class FuelTankResponse {
    private Long id;
    private FuelType fuelType;
    private BigDecimal capacityGallons;
    private BigDecimal currentLevelGallons;
    private BigDecimal fillPercentage;
    private LocalDateTime lastUpdated;
}

File: FuelControl/app/src/main/java/co/edu/unipiloto/fuelcontrol/api/requests/FuelTankResponse.kt
package co.edu.unipiloto.fuelcontrol.api.requests

data class FuelTankResponse(
    val id: Long,
    val fuelType: String,
    val capacityGallons: Double,
    val currentLevelGallons: Double,
    val fillPercentage: Double,
    val lastUpdated: String
)